cloudflare/src/endpoints/workers/mod.rs

```diff
@@ -3,7 +3,9 @@ use crate::framework::response::ApiResult;
 use serde::Deserialize;
 
 mod create_route;
+pub mod create_secret;
 mod delete_route;
+pub mod delete_secret;
```

Comment:
i know  you said you had given it  a try earlier, but i'd like to do the same `pub use` as we did with routes endpoints. the  reason being that the way this will look in client code is:

``` rust
// we prefer this
use cloudflare::endpoints::workers::{CreateRoute, CreateRouteParams}
// over this
use cloudflare::endpoints::workers::create_secret::{CreateSecret, CreateSecretParams}
```

Reply:
I try to do that but it kept complaining it was private or not defined. Could you point me to how you used it in wrangler routes code?